src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/Card/CardPaymentsResource.cs

```diff
@@ -45,15 +45,14 @@ public async Task<ICardPaymentRecurResponse> Create(CardPaymentRecurRequest paym
             return new CardPaymentRecurResponse(cardPaymentResponseDto, HttpClient);
         }
 
-        public async Task<ICardPaymentVerifyResponse> Create(CardPaymentVerifyRequest paymentRequest, PaymentExpand paymentExpand = PaymentExpand.All)
+        public async Task<ICardPaymentRecurResponse> Create(CardPaymentVerifyRequest paymentRequest, PaymentExpand paymentExpand = PaymentExpand.All)

```

Comment:
Is it not a bit strange that creating a `Verify` payment returns a `Recur` response?

Reply:
True, maybe should be `ICardPaymentResponse` as well instead.

Reply:
As long as it makes sense and is consistent with the other `Create` operations, that should probably be fine.